src/test/java/group1/mips_simulator/UtilityTest.java

```diff
@@ -44,4 +46,11 @@ void shortToBinaryString_Pretty() {
         assertEquals(Utility.shortToBinaryString_Pretty((short) 15, (short) 16), "0000_0000_0000_1111");
     }
 
+    @Test
+    void instructionToShort() {
+//        Field f1 = new Field(0);
+//        Instruction i1 = new Instruction();
+//        assertEquals(Utility.instructionToShort( ));
```

Comment:
Why are these commented out?

Reply:
Oh I see, you just moved them to `InstructionTest.java`. Ok! Looks good

Reply:
ah yep, ill delete those from the test file